src/page-weekly-snapshot/LocaleSummaryTable.tsx

```diff
@@ -0,0 +1,368 @@
+import React from "react";
+import styled from "styled-components";
+
+import Colors from "../design-system/Colors";
+import { Column, PageContainer } from "../design-system/PageColumn";
+import { useFacilities } from "../facilities-context";
+import {
+  caseBracketKeys,
+  deathBracketKeys,
+  incarceratedPopulationKeys,
+} from "../impact-dashboard/EpidemicModelContext";
+import { formatThousands } from "../impact-dashboard/ImpactProjectionTable";
+import { LocaleData, useLocaleDataState } from "../locale-data-context";
+import { Facility } from "../page-multi-facility/types";
+
+const Table = styled.table`
+  color: ${Colors.black};
+  text-align: left;
+  width: 100%;
+  margin-top: 10px;
+`;
+
+const HorizontalRule = styled.hr<{ width?: string; marginLeft?: string }>`
+  border-color: ${Colors.black};
+  width: ${(props) => props.width || "100%"};
+  margin-left: ${(props) => props.marginLeft || "0"};
+`;
+
+const TableHeadingCell = styled.td`
+  font-family: "Libre Franklin";
+  font-weight: bold;
+  font-size: 11px;
+  line-height: 13px;
+  vertical-align: middle;
+`;
+
+const LeftHeading = styled.div`
+  text-align: left;
+`;
+
+const TextContainerHeading = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 10px;
+`;
+
+const TextContainer = styled.div`
+  width: 100%;
+  margin: 15px 0 15px;
+  display: flex;
+  justify-content: space-between;
+  align-items: baseline;
+  color: ${Colors.black};
+`;
+
+const Right = styled.div`
+  text-align: right;
+`;
+
+const Left = styled.div`
+  text-align: left;
+  font-size: 24px;
+  font-family: "Libre Baskerville";
+`;
+
+const BorderDiv = styled.div`
+  border-top: 2px solid ${Colors.black};
+`;
+
+const TableCell = styled.td<{ label?: boolean }>`
+  font-size: 11px;
+  line-height: 200%;
+  text-align: "left";
+  width: ${(props) => (props.label ? "200px" : "auto")};
+`;
+
+type StateMetrics = {
+  stateName: string;
+  casesPerCapita: number | undefined;
+  deathsPerCapita: number | undefined;
+};
+
+type incarceratedData = {
+  incarceratedData: number;
+  hasData: boolean;
+};
+
+const POPULATION_SIZE = 100000;
+const NUM_STATES = 50;
+
+const VALUE_MAPPING = {
+  deaths: deathBracketKeys,
+  cases: caseBracketKeys,
+  population: incarceratedPopulationKeys,
+};
+
+function formatOrdinal(value: number) {
+  let suffix = ["th", "st", "nd", "rd"],
+    rem = value % 100;
+  return value + (suffix[(rem - 20) % 10] || suffix[rem] || suffix[0]);
+}
+
+function getPerCapita(
+  numerator: number | undefined,
+  denominator: number | undefined,
+) {
+  let result = 0;
+  if (numerator && denominator) {
+    result = Math.round((numerator / denominator) * POPULATION_SIZE);
+  }
+  return result;
+}
+
+function getAllStateData(localeData: LocaleData, stateNames: string[]) {
+  const stateMetrics: StateMetrics[] = [];
+  // D.C. isn't a state!
+  stateNames = stateNames.filter((item) => item !== "District of Columbia");
```

Comment:
this is true .... do we definitely want to exclude it from these metrics? Does it break anything to leave it in?

Reply:
Ideally we would leave it in but if we can't because it breaks something, then let's let @sychang know so she can prioritize future support.

Reply:
I actually did ask @sychang about this, and it could just potentially make the rankings weird because including D.C. means we are ranking out of 51 "states." More specifically, leaving D.C. in might mean someone sees a "51st out of 50" ranking

Reply:
Got it. Thanks for the context, @neeharmb